lib/Migration/Version21002Date20250509153510.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Talk\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version21002Date20250509153510 extends SimpleMigrationStep {
```

Comment:
```suggestion
class Version21001Date20250509153510 extends SimpleMigrationStep {
```

gonna make it part of 21.1 the version is XYYY when Talk is X.Y.Z